packages/styles/src/components/alert.scss

```diff
@@ -72,6 +72,14 @@
   font-size: alert.$alert-font-size;
 }
 
+.alert-icon {
+  @extend %notification-icon;
+
+  height: alert.$alert-icon-size;
+  width: alert.$alert-icon-size;
+  left: alert.$alert-padding-x;
+}
```

Comment:
I added a specific class in case the user wants to use an icon in the alert message:
![image](https://github.com/swisspost/design-system/assets/33580481/a89373ed-384d-4df4-a6ad-af07357d702d)